### PR TITLE
Add Availity Essentials

### PR DESCRIPTION
**Pull request by contextnerror**

### CODE REVIEW

File: entries/a/availity.com.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "Availity Essentials": {
+    "domain": "availity.com",
+    "tfa": [
+      "sms",
+      "call",
+      "totp"
+    ],
```
Comment (hkamran80) [inline entries/a/availity.com.json:8]: Do they also support U2F?

> 2-step enrollment requires you to register your preferred authentication method through your choice
of text, voice, mobile authenticator app (i.e., Google Authenticator), or a hard token, also known as a
security token or authorization token.

**Comment by hkamran80** [comment]
Do they also support U2F?

> 2-step enrollment requires you to register your preferred authentication method through your choice
of text, voice, mobile authenticator app (i.e., Google Authenticator), or a hard token, also known as a
security token or authorization token.

**Comment by contextnerror** [comment]
No, hardware tokens are OAuth only.

**Comment by hkamran80** [comment]
Thanks!

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**